bitnami/argo-workflows/Chart.yaml

```diff
@@ -0,0 +1,38 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 3.1.10
+dependencies:
+  - condition: controller.persistence.postgresql.enabled
+    name: postgresql
+    repository: https://charts.bitnami.com/bitnami
+    version: 10.x.x
+  - condition: controller.persistence.mysql.enabled
```

Comment:
Use the same standard as with other charts. mysql.enabled and postgresql.enabled.

I see that it is using mysql and most of ours are using mariadb. Any reason for this?

Reply:
The reason about using MySQL instead of MariaDB is that the upstream ones references MySQL not MariaDB, but in any case, this is a feature I added that is not in the upstream one, so I guess we can configure it however we want.

Reply:
I moved to the standard making a big refactor of the configuration, about using mariadb instead of MySQL the change should be trivial, but there is no reference to mariadb in the docs:
```
configure a Postgres or MySQL (>= 5.7.8) database under 
```